Getting-Started-With-Orleans/Grain-Persistence.md

```diff
@@ -297,4 +325,35 @@ Any other failure conditions from a write operation _should_ cause the write `Ta
 
 Individual storage providers should decide how best to store grain state – blob (various formats / serialized forms) or column-per-field are obvious choices.
 
-The basic storage provider for Azure Table encodes state data fields into a single table column using Orleans binary serialization.
\ No newline at end of file
+The basic storage provider for Azure Table encodes state data fields into a single table column using Orleans binary serialization.
+
+
+## ADO.NET Persistence Rationale <a name="ADONETPersistenceRationale"></a>
+
+The principles for ADO.NET backed persistence storage are:
+
+1. Keep business critical data safe an accessible while data, the format of data and code evolve.
+2. Take advantenge of vendor and storage specific functionality.
+
+In practice this means adhering to [ADO.NET implementation goals](../Runtime-Implementation-Details/Relational-Storage)
+and some added twists in ADO.NET specific storage provider that allow evolving the shape of the data in the storage.
+
+In addition to the usual storage provider capabilities, the ADO.NET provider has built-in capability to
```

Comment:
I'll take this and your other notes and improve them. It might be on Monday evening when I have an improved set. Just to give a bit of heads up if I fall silent for a few days.